pkgs/misc/uboot/default.nix

```
@@ -139,6 +139,12 @@ in {
     filesToInstall = ["u-boot-sunxi-with-spl.bin"];
   };
 
+  ubootA20OlinuxinoLime2 = buildUBoot {
```

```suggestion
  ubootA20OlinuxinoLime2EMMC = buildUBoot {
```

The attribute name should represent the defconfig name correctly. I missed that on my first pass. No, it's not needed to add the "non-eMMC" variant of the defconfig.

---

Just noticed, [ubootOlimexA64Olinuxino](https://github.com/NixOS/nixpkgs/blob/ddafcd3f7a01110e0308c984f9437bae2ae17e6b/pkgs/misc/uboot/default.nix#L277) is missing the "EMMC" too.